openedx/adg/common/lib/zoom_client/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+This package contains all the communication with Zoom via the APIs.
+
+Zoom client depends on ZOOM_API_KEY and ZOOM_SECRET_KEY. It uses a python package named zoomus to handle all the
+communications with Zoom. It is recommended to mock zoom client while writing tests so that the tests does
+not depend on external entity. It also means that the ZOOM_API_KEY and ZOOM_SECRET_KEY won't be needed for tests.
```

Comment:
Why do we need to add the part about testing here? Is it relevant here?

Reply:
Looked into other clients and it was added there so just followed similar format.